src/garage/tf/algos/reps.py

```diff
@@ -15,7 +15,8 @@
 from garage.tf.optimizers import LbfgsOptimizer
 
 
-class REPS(BatchPolopt):
+# pylint: disable=differing-param-doc, differing-type-doc
+class REPS(BatchPolopt):  # noqa: D416
```

Comment:
what is D416?

Reply:
This is also because flake8 cannot recognize the refences section in docstring.